src/libs/mysqlHandler.go

```diff
@@ -961,11 +961,10 @@ func updateOrInsertKubearmorLogMySQL(db *sql.DB, kubearmorlog types.KubeArmorLog
 
 	if err == nil && rowsAffected == 0 {
 
-		updateQueryString := `(cluster_name,host_name,namespace_name,pod_name,container_id,container_name,
-		uid,type,source,operation,resource,labels,data,category,action,start_time,
-		updated_time,result,total) values(?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`
```

Comment:
why are we removing log type?

Reply:
I think we do not need Log Type
As we are handling/see difference in pod names 
vm/container have vm-podname and container-podname